docs/scalardb-supported-databases.md

```diff
@@ -1,31 +1,141 @@
-| ScalarDB 3.7 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.6 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.5 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.4 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.3 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.2 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.1 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  |
-| ScalarDB 3.0 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  Not tested  |  Not tested  |  Not tested  |  ✅  |  ✅  |  ✅  |  Not tested  |  Not tested  |  ✅  |  ✅  |  |
-| ScalarDB 2.4 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.3 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.2 |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.1 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.0 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
+# Supported Databases
+
+ScalarDB supports the following databases and their versions.
+
+## Amazon DynamoDB
+
+| Version | Amazon DynamoDB |
+|:------------------|:---------------|
+| **ScalarDB 3.10** | ✅ |
+| **ScalarDB 3.9** | ✅ |
+| **ScalarDB 3.8** | ✅ |
+| **ScalarDB 3.7** | ✅ |
+| **ScalarDB 3.6** | ✅ |
+| **ScalarDB 3.5** | ✅ |
+| **ScalarDB 3.4** | ✅ |
+
+## Apache Cassandra
+
+{% capture notice--info %}
+**Note**
+
+For requirements when using Cassandra or Cassandra-compatible databases, see [Cassandra or Cassandra-compatible database requirements](requirements/#cassandra-or-cassandra-compatible-database-requirements).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+| Version | Apache Cassandra 4.1 | Apache Cassandra 4.0 | Apache Cassandra 3.11 | Apache Cassandra 3.0 | Apache Cassandra 2.2 |
```

Comment:
I couldn't find any previous PRs or discussions about ScalarDB supporting Cassandra 4.0 or 4.1. Are those versions not officially supported? If not, I think we should add an ❌ to those versions in this table. 

Or, if we know that those versions work with ScalarDB but we haven't officially tested them, we can mention "Not tested". 

What do you think about that?

Reply:
(Posted [a related comment](https://github.com/scalar-labs/scalardb/pull/1012/files#r1302364828))

Reply:
Yeah, currently, we don't support Cassandra 4.0 and 4.1.

Reply:
Got it. To reduce potential support inquiries, I've marked Cassandra 4.0 and 4.1 as unsupported.